kpops/components/base_components/base_defaults_component.py

```diff
@@ -238,8 +238,9 @@ def defaults_from_yaml(path: Path, key: str) -> dict:
     value = content.get(key)
     if value is None:
         return {}
+    default_path = path.resolve().relative_to(Path.cwd())
```

Comment:
there's no need for `resolve()`, path is already absolute. also, could you move this to the log statement